subt_ign/src/GameLogicPlugin.cc

```diff
@@ -720,6 +787,7 @@ void GameLogicPluginPrivate::OnBreadcrumbDeployRemainingEvent(
         << "  robot: " << name << std::endl;
 
       this->LogEvent(stream.str());
+      this->PublishRobotEvent("max_breadcrumb_deploy", name);
```

Comment:
Can the `max_breadcrumb_deploy` callback be utilized to _avoid_ publishing `breadcrumb_deploy` if the breadcrumb did not actually deploy rather than publishing both `breadcrumb_deploy` and `max_breadcrumb_deploy` events when nothing happened?

Reply:
Done: 2ccac25f001be9ddfbac4fcd0410f3415dd30011